package.json

```diff
@@ -29,9 +31,9 @@
     "@types/minimist": "^1.2.0",
     "@types/node": "^17.0.22",
     "form-data": "^4.0.0",
-    "husky": "^7.0.0",
+    "husky": "^8.0.1",
     "jest": "^27.0.5",
-    "lint-staged": "^12.0.2",
+    "lint-staged": "^12.5.0",
```

Comment:
There is no need to configure `lint-staged`? It will just detect the Prettier config?